File: src/test/java/com/_0d0a/QuasarIncreasingEchoTest.java
package com._0d0a;

import static org.junit.Assert.*;
import org.junit.Test;
import com._0d0a.QuasarIncreasingEchoApp;

import static org.hamcrest.CoreMatchers.*;

/**
 * Increasing-Echo Quasar Text
 *
 * @author circlespainter
 */
public class QuasarIncreasingEchoTest {
    @Test
    public void test() throws Exception {
        assertThat(QuasarIncreasingEchoApp.doAll(), is(10));
    }
}

File: src/test/kotlin/com/_0d0a/QuasarIncreasingEchoTest.kt
package com._0d0a;

import org.junit.Assert.*
import org.junit.Test
import com._0d0a.QuasarIncreasingEchoApp

import org.hamcrest.CoreMatchers.*

/**
 * Increasing-Echo Quasar Text
 *
 * @author circlespainter
 */
public class QuasarIncreasingEchoTest {
    @Test
    public fun test() {
        assertEquals(QuasarIncreasingEchoApp.doAll(), 10);
    }
}